ios/PolyPodApp/PolyPod/RuntimeInfo.swift

```diff
@@ -11,4 +11,45 @@ struct RuntimeInfo {
         let buildNumber = info["CFBundleVersion"] ?? "Unknown"
         return "\(marketingVersion) (\(buildNumber))"
     }
+    
+    static var isProduction: Bool {
+        if isDebug || isRunningInTestFlightEnvironment {
+            return false
+        }
+        
+        return true
+    }
+    
+    static var isDebug: Bool {
+        #if DEBUG
+            return true
+        #else
+            return false
+        #endif
+    }
+    
+    private static var isRunningInTestFlightEnvironment: Bool {
+        if isAppStoreReceiptSandbox && !hasEmbeddedMobileProvision {
```

Comment:
Why not just `return isAppStoreReceiptSandbox && !hasEmbeddedMobileProvision`?

Reply:
Sure, was originally playing with different combinations. Did a cleanup also for the rest.